atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java

```diff
@@ -39,17 +38,16 @@ public Integer call() {
         }
```

Comment:
Discussed offline: behavior expected is to log stacktrace and print to System.err.